### ISSUE TITLE
Getting started

### ISSUE THREAD

**Issue by Harry352** [ct-Open-Source/team-container#73]
I'm sorry can't passing the frist step..
After helm install setup team-setup --values values-setup.yaml apears this:
Error: INSTALLATION FAILED: failed to install CRD crds/traefik-crds.yaml: unable to recognize "": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1beta1"
Ionos
Ubuntu 20.4
thankfull for hints ;)

New day  - same issue now on 
Contabo
Ubuntu 20.4.3

bg Harry

**Comment by ulrich4** [comment]
Dear Harry,

i'm fighting against the troubles, too.  My router/Traefik has gone after a reinstall....

Maybe this will help you?

https://doc.traefik.io/traefik/providers/kubernetes-crd/
-----------------------------------------------------------------------
Deprecated apiextensions.k8s.io/v1beta1 CRD
The apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in Kubernetes v1.16+ and will be removed in v1.22+.

For Kubernetes v1.16+, please use the Traefik apiextensions.k8s.io/v1 CRDs instead.
-----------------------------------------------------------------------



Best wishes!
Ulrich

**Comment by mshoff** [comment]
I also came across this. I have tried to apply these changes. You also need to change the names and add schema data (see treafik docu). After the changes the `install.sh` runs without error and the pods are running.

But unfortunately treafik still does not work. A look into the logs shows me the following messages:

```log
E0203 08:19:06.847075       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.22.1/tools/cache/reflector.go:167: Failed to watch *v1alpha1.MiddlewareTCP: failed to list *v1alpha1.MiddlewareTCP: middlewaretcps.traefik.containo.us is forbidden: User "system:serviceaccount:default:traefik-ingress-controller" cannot list resource "middlewaretcps" in API group "traefik.containo.us" at the cluster scope
E0203 08:19:06.850752       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.22.1/tools/cache/reflector.go:167: Failed to watch *v1alpha1.ServersTransport: failed to list *v1alpha1.ServersTransport: serverstransports.traefik.containo.us is forbidden: User "system:serviceaccount:default:traefik-ingress-controller" cannot list resource "serverstransports" in API group "traefik.containo.us" at the cluster scope
[...]
```

**Comment by ulrich4** [comment]
Yes, my pods are running, but i get a "404 page not found" from nginx :(  
The acme.json-File is empty. Maybe it is a problem with Let's encryped? 

BTW. I have the same error messages in the log....

**Comment by mshoff** [comment]
I think further adjustments are necessary here. I stopped at this point and switched to [NextCloud All In One](https://github.com/nextcloud/all-in-one).

**Comment by ulrich4** [comment]
In this moment I have success!
Compare the content of 01-role.yaml with the (new) traefik-crds.yaml...

01-role.yaml:
--------------------------------------------------------------------------------------------
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: traefik-ingress-controller
rules:
  - apiGroups:
      - ""
    resources:
      - services
      - endpoints
      - secrets
    verbs:
      - get
      - list
      - watch
  - apiGroups:
      - extensions
    resources:
      - ingresses
    verbs:
      - get
      - list
      - watch
  - apiGroups:
      - extensions
    resources:
      - ingresses/status
    verbs:
      - update
  - apiGroups:
      - traefik.containo.us
    resources:
      - middlewares
      - middlewaretcps
      - ingressroutes
      - traefikservices
      - ingressroutetcps
      - ingressrouteudps
      - tlsoptions
      - tlsstores
      - serverstransports
    verbs:
      - get
      - list
      - watch
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: traefik-ingress-controller

roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: traefik-ingress-controller
subjects:
  - kind: ServiceAccount
    name: traefik-ingress-controller
    namespace: default
---
apiVersion: v1
kind: ServiceAccount
metadata:
  namespace: default
  name: traefik-ingress-controller
--------------------------------------------------------------------------------------------

**Comment by Harry352** [comment]
> I think further adjustments are necessary here. I stopped at this point and switched to [NextCloud All In One](https://github.com/nextcloud/all-in-one).

I stopped too and split the project 1.) to a managed nextcloud and 2.) to a managed bbb Server, because I had no more time to spend...
